### PR TITLE
Add support for starting all cluster services

### PR DESCRIPTION
**Pull request by dimaspivak**

### CODE REVIEW

File: ambariclient/models.py [hunk 1]
```diff
@@ -907,6 +907,21 @@ def execute_action(self, action, context, parameters=None, hosts=None):
         }))
         return self.request
 
+    def start(self):
```
Comment (jimbobhickville) [inline ambariclient/models.py]: This should probably be on the `ClusterService` model so you can do `cluster(name).services.start()`

Comment (jimbobhickville) [reply ambariclient/models.py]: Wait, that may not be right. I haven't looked at this code in a while.

Comment (jimbobhickville) [reply ambariclient/models.py]: You would need to create a `ClusterServiceCollection` with this method, similar to how `HostComponentCollection` does for `HostComponent`, and then set `collection_class = ClusterServiceCollection` on `ClusterService`.

Comment (dimaspivak) [reply ambariclient/models.py]: Ah, good call. Let me fix this up and push another attempt. Thanks for the review! :)

**Pull request merged**